edx_exams/apps/access/views.py

```diff
@@ -0,0 +1,63 @@
+"""
+Exam Access Ticket Views
```

Comment:
Question: I added an access package to add views/endpoints for access ticket specifically. Should this go somewhere else?

Reply:
I think you might be better off simply including this in the api app. Access is only going to ever include this one relatively small endpoint.